packages/platforms/delta/arm64/tn48m/src/arm64_delta_tn48m/module/src/psui.c

```diff
@@ -33,6 +33,7 @@ static int _psu_event(void *e, int ev);
 static plat_psu_t plat_tn48m_psus[] = {
     [PLAT_PSU_ID_1] = {
         .name = "PSU",
+	.type = PLAT_PSU_TYPE_AC,
```

Comment:
Please indent the line with spaces as done in the other lines. Also present at other places below.